---
title: Polar antibiotic discovery,uncovering the biomedical potential of rare actinomycetes isolated from the Arctic and Antarctic.

summary: To work through a drug discovery data analysis pipeline to assess the potential for Polar Actinobacteria to produce novel antibiotics
tags:
- Bioinformatics
date: "09/2021-now"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: Photo by rawpixel on Unsplash
  focal_point: Smart

#links:
#- icon: twitter
#  icon_pack: fab
#  name: Follow
 # url: https://twitter.com/georgecushen
#url_code: ""
#url_pdf: ""
#url_slides: ""
#url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
#slides: example
---

  Microbes living at the extremes are testing out possibilities and carving out a niche to survive. These microbial pioneers are pushing limits; as a microbiologist the questions are exciting, challenging and represent a fascinating world of discovery.
  Antarctic and Arctic ecosystems are some of the coldest, windiest and driest place on Earth,yet bacterial species have been evolving to inhabit the extreme environmental niches. Through this data analysis project, you will be uncovering the biological and chemical diversity of Actinobacteria isolated from marine sediment. These bacterial pioneers have been trapped in sediment, up to 3000 meters below the Ocean surface for decades, representing an incredible opportunity for assessing their antibiotic potential! In this project, you will work through a drug discovery pipeline, first you will investigate the phylogeny of your strain (16S rRNA gene sequences), this will provide evolutionary species information. You will then be provided with the full genome sequence of your strain. You will mine your bacterial genome sequence (using AntiSMASH) for biosynthetic gene clusters (which may encode antibiotics), in addition to further phylogenetic work, using AutoMLST. The chemistry produced by these strains will also be assessed using mass spectrometry data and comparative metabolomics platforms. Lastly, you will be provided with antibacterial assay results of your strain against a range of clinically relevant pathogens. Through these combined datasets and bioinformatic methods, you will be able to assess the potential of these strains to produce novel antibiotics. 
  Project aim
  To work through a drug discovery data analysis pipeline to assess the potential for Polar Actinobacteria to produce novel antibiotics.
  Techniques
  Phylogenetic analyses (16S rRNA gene and whole genome), genome assembly, genome mining for biosynthetic gene clusters (antibiotic potential), comparative metabolomics using mass spectrometry data, assessing antibiotic activity of microbial strains against a range of clinical pathogens.
